packages/@okta/vuepress-site/docs/reference/api/inline-hooks/index.md

```diff
@@ -244,11 +244,11 @@ curl -v -X GET \
 
 The submitted Inline Hook properties will replace the existing properties after passing validation. Note that some properties are immutable and cannot be updated. Refer to the description of each property in the [Inline Hook object](#inline-hook-object) table for information.
 
-##### Response Parameters
+##### Response parameters
 
 The response is an [Inline Hook object](#inline-hook-object) representing the updated Inline Hook.
```

Comment:
The response is an [Inline Hook object](#inline-hook-object) that represents the updated Inline Hook.